streams/runners/storm/actions/src/main/java/com/hortonworks/streamline/streams/actions/storm/topology/StormTopologyActionsImpl.java

```diff
@@ -127,6 +136,35 @@ public void init (Map<String, String> conf) {
         }
         File f = new File (stormArtifactsLocation);
         f.mkdirs();
+
+        setupSecuredStormCluster(conf);
```

Comment:
The instance is created per each namespace (environment) so the configuration is specific to one of Storm cluster, other instance will have other configuration specific to other Storm cluster.